package.json

```diff
@@ -87,6 +83,7 @@
   },
   "license": "MIT",
   "scripts": {
+    "preinstall": "npm install node-pre-gyp node-pre-gyp-github",
```

Comment:
Are you sure? From the readme (https://github.com/bchr02/node-pre-gyp-github) it looks like it is only for posting binaries. As `node-pre-gyp` itself downloads over http, it handles that fine.